packages/node_modules/@webex/widget-meet/src/components/call-active/index.js

```diff
@@ -10,9 +10,10 @@ const ActiveMeeting = ({adapter, meetingId}) => (
   <WebexDataProvider adapter={adapter}>
     <div className={classNames(styles.callContainer, 'call-container')}>
       <WebexMeetingControls className={styles.callControls} meetingID={meetingId}>
-        <WebexMeetingControl type="mute-audio" />
-        <WebexMeetingControl type="mute-video" />
-        <WebexMeetingControl type="leave-meeting" />
+        <WebexMeetingControl type="mute-audio" meetingID={meetingId} />
+        <WebexMeetingControl type="mute-video" meetingID={meetingId} />
+        <WebexMeetingControl type="share-screen" meetingID={meetingId} />
+        <WebexMeetingControl type="leave-meeting" meetingID={meetingId} />
```

Comment:
meetingID is now a required prop, which it wasnt before
This seems like a breaking change of sorts and couldnt find anything in the changelog that said it was
Was alerted when i scrolled through the console log

Reply:
yeah, that changed. Thanks for catching it!